src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.model.dto.request;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class UserRequestDto {
+    @Email(message = "Not valid email")
+    @NotNull(message = "Please enter your email")
+    @Size(min = 4)
+    private String email;
+    @NotNull(message = "Please enter your password")
+    @Size(min = 6)
```

Comment:
validation is next task scope
![image](https://user-images.githubusercontent.com/20326858/233337097-702148be-3689-42c5-b177-70e982718832.png)